backends/backends.c

```diff
@@ -337,9 +337,8 @@ void *backends_main(void *ptr) {
             backend_request_formatter = format_dimension_stored_json_plaintext;
 
     }
-#if HAVE_KINESIS
     else if (!strcmp(type, "kinesis") || !strcmp(type, "kinesis:plaintext")) {
-
+#if HAVE_KINESIS
```

Comment:
Same question with above actually, why kinesis stuff moving around?
Perhaps i should see the whole file to understand it?

Reply:
This one was needed to print an error message if a user set kinesis backend but its support hadn't been compiled.